### PR TITLE
Add/openai placeholder prompts

### LINKED ISSUE #28325: [Jetpack AI] Suggested prompts
It would be neat to suggest prompts in the placeholder, so we can nudge people to write good ones specifically for DALL-e.

1. Find good prompts. At least 10. test them on Dall-e. Here is some inspiration: https://prompthero.com/dall-e-prompts
2. Create a file with them
3. On render, randomly select one and put as the placeholder in the AI image block

### PR DESCRIPTION
**Pull request by millerf**
Fixes https://github.com/Automattic/jetpack/issues/28325

## Proposed changes:

Add prompt suggestions for AI Image

## Testing instructions:

- Deploy this to your sandbox.
- Sandbox a site
- Create an AI Image
- You should see a placeholder text. Test that it behaves accordingly.
- When clicking on submit it prompts with the placeholder
- 
Screenshot 2023-01-13 at 11 37 38

## Examples of the prompts results

Screenshot 2023-01-13 at 17 20 56
 Screenshot 2023-01-13 at 17 21 02
Screenshot 2023-01-13 at 17 21 10
Screenshot 2023-01-13 at 17 21 17

Screenshot 2023-01-13 at 17 20 48

### CODE REVIEW

File: projects/plugins/jetpack/extensions/blocks/ai-image/edit.js [hunk 1]
```diff
@@ -14,6 +14,31 @@ import { useSelect, useDispatch } from '@wordpress/data';
...
 import { __ } from '@wordpress/i18n';
 
+function getRandomItem( arr ) {
+	// get random index value
+	const randomIndex = Math.floor( Math.random() * arr.length );
+	return arr[ randomIndex ];
+}
+
+const PLACEHOLDER = getRandomItem( [
+	'earth reviving after human extinction, a new beginning, nature taking over buildings, animal kingdom, harmony, peace, earth balanced --version 3 --s 1250 --uplight --ar 4:3 --no text, blur',
```
Comment (jeherve) [inline projects/plugins/jetpack/extensions/blocks/ai-image/edit.js]: Do we want those to be translated?

Comment (artpi) [reply projects/plugins/jetpack/extensions/blocks/ai-image/edit.js]: Frankly no, I don't think so. While I expect them to work in other languages, the translators would have to also check if these are producing decent results as they would most likely be differnt. Let's have them separate for now

File: projects/plugins/jetpack/extensions/blocks/ai-image/edit.js [hunk 2]
```diff
@@ -69,14 +94,16 @@ export default function Edit( { attributes, setAttributes, clientId } ) {
 		};
 	}, [] );
 
-	const submit = () =>
+	const submit = () => {
+		setErrorMessage( '' );
 		getImagesFromOpenAI(
-			prompt,
+			prompt.trim() === '' ? PLACEHOLDER : prompt,
```
Comment (artpi) [inline projects/plugins/jetpack/extensions/blocks/ai-image/edit.js]: I have some doubts about that, but I'll allow it.

File: projects/plugins/jetpack/extensions/blocks/ai-image/edit.js [hunk 3]
```diff
@@ -14,6 +14,31 @@ import { useSelect, useDispatch } from '@wordpress/data';
 import { useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
+function getRandomItem( arr ) {
+	// get random index value
+	const randomIndex = Math.floor( Math.random() * arr.length );
+	return arr[ randomIndex ];
+}
+
+const PLACEHOLDER = getRandomItem( [
```
Comment (artpi) [inline projects/plugins/jetpack/extensions/blocks/ai-image/edit.js]: Let's have this list in seperate file

**Review by artpi — CHANGES_REQUESTED** [review]
I left some comments but we're gonna need to test each prompt to make sure its producing decent results in DALL-e

File: projects/plugins/jetpack/extensions/blocks/ai-image/edit.js [hunk 4]
```diff
@@ -14,6 +14,25 @@ import { useSelect, useDispatch } from '@wordpress/data';
...
```
Comment (artpi) [inline projects/plugins/jetpack/extensions/blocks/ai-image/edit.js]: By having randomness here and not in Render, when you insert the block twice into the same post, you will have the same prompts.
So it looks like there is only one prompt.

File: projects/plugins/jetpack/extensions/blocks/ai-image/edit.js (hunk 3 above)
Comment (artpi) [reply projects/plugins/jetpack/extensions/blocks/ai-image/edit.js]: Yeah different file would be easier to update over time

File: projects/plugins/jetpack/extensions/blocks/ai-image/edit.js (hunk 4 above)
Comment (millerf) [reply projects/plugins/jetpack/extensions/blocks/ai-image/edit.js]: Good point...

File: projects/plugins/jetpack/extensions/blocks/ai-image/edit.js (hunk 2 above)
Comment (millerf) [reply projects/plugins/jetpack/extensions/blocks/ai-image/edit.js]: :)

File: projects/plugins/jetpack/extensions/blocks/ai-image/edit.js (hunk 3 above)
Comment (millerf) [reply projects/plugins/jetpack/extensions/blocks/ai-image/edit.js]: Done

File: projects/plugins/jetpack/extensions/blocks/ai-image/edit.js (hunk 4 above)
Comment (millerf) [reply projects/plugins/jetpack/extensions/blocks/ai-image/edit.js]: Done.

**Review by artpi — DISMISSED** [review]

**Review by jeherve — COMMENTED** [review]
Would it be possible to have the prompt input field be a bit more dynamic, and maybe full width of its container, to avoid layout issues with long prompts like this one:

Screenshot 2023-01-16 at 11 56 00

**Comment by millerf** [comment]
> Would it be possible to have the prompt input field be a bit more dynamic, and maybe full width of its container, to avoid layout issues with long prompts like this one:
> 
> Screenshot 2023-01-16 at 11 56 00

Yes, this is actually part of #28323 and is being worked on.

**Comment by millerf** [comment]
@jeherve What do you mean exactly by "dynamic?

**Comment by jeherve** [comment]
> @jeherve What do you mean exactly by "dynamic?

Mostly more flexible, so it can accomodate longer prompts as well. But if you're working on that already, all good :)

**Comment by artpi** [comment]
> Would it be possible to have the prompt input field be a bit more dynamic, and maybe full width of its container, to avoid layout issues with long prompts like this one:

You mean make the textbox grow when typing, correct?

**Comment by jeherve** [comment]
Yup, exactly.

**Review by artpi — APPROVED** [review]
I approve these changes

**Comment by millerf** [comment]
@jeherve Do you mind accepting it again (I had to merge trunk)...

**Review by jeherve — APPROVED** [review]

**Pull request merged**